taskRequests/details/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" href="/images/index.ico" type="image/x-icon" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;400;700;800&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./style.css" />
+
+    <title>Task Requests | Real Dev Squad</title>
+    <script src="/helpers/loadENV.js"></script>
+    <script src="/taskRequests/util.js"></script>
+    <script src="./script.js" defer></script>
+  </head>
+  <body>
+    <div class="header">
+      <div class="header__contents">
+        <img
+          src="/taskRequests/assets/RDSLogo.png"
+          alt="RDS logo"
+          height="48"
+          width="48"
+        />
+        <a href="/" class="header__contents__navlink">Home</a>
+      </div>
+    </div>
+    <div class="container">
+      <div id="task-request-details" class="taskRequest">
+        <div class="taskRequest__skeleton">
+          <div class="taskRequest__skeleton__title skeleton"></div>
+          <div class="taskRequest__skeleton__subtitle skeleton"></div>
+        </div>
+      </div>
+
+      <div id="task-details" class="task">
+        <div class="task__skeleton">
+          <div class="task__skeleton__title skeleton"></div>
+          <div class="task__skeleton__details skeleton"></div>
+          <div class="task__skeleton__details skeleton"></div>
+          <div class="task__skeleton__description skeleton"></div>
+          <div class="task__skeleton__description skeleton"></div>
+          <div class="task__skeleton__description skeleton"></div>
+          <div class="task__skeleton__description skeleton"></div>
+        </div>
+      </div>
+
+      <div class="requestors">
+        <div class="requestors__container">
+          <h4 class="requestors__container__title">Requestors</h4>
+          <ul class="requestors__container__list" id="requestors-details"></ul>
+          <ul
+            class="
+              requestors__container__list requestors__container__list__skeleton
+            "
+          >
+            <li class="skeleton"></li>
+            <li class="skeleton"></li>
+            <li class="skeleton"></li>
+            <li class="skeleton"></li>
```

Comment:
NIT - instead we could have used this via js file here.

Reply:
adding it via js, will create more complexity as we would have to create and track the elements in the script as well. Currently we can just add the hidden class in the parent, which makes it a lot simpler.